pennylane/operation.py

```diff
@@ -86,15 +87,15 @@
   operation on the vector of quadrature operators :math:`\mathbf{r}` for the given parameter
   values.
 
-* For Gaussian CV Expectations this method should return a real vector (first-order observables)
+* For Gaussian CV Observables this method should return a real vector (first-order observables)
```

Comment:
Might be some ambiguity here, since it appears we're already allowing variances (in covariance matrix) to be returned as expvals.
Maybe we'll need a `cov` measurement type?

Reply:
I'm not sure I fully follow?

Reply:
There are a couple measurement types in the CV case which allow you to return expectation values of second-order quadrature operators. But second order expectations values are (more or less) also (co)variances. So we should be careful to think about how we return things in these cases when we have both `expval` and `var` implemented.

Reply:
Should we revisit once the `variance` PR is ready?

Reply:
Pretty much, but we should revisit while actually doing the variance PR :smile: